### ISSUE TITLE
Importing styles creates unnecessary subfolders

### ISSUE THREAD

**Issue by AlexisDoroszkiewicz** [tokens-studio/figma-plugin#1468]
**Describe the bug**

When importing styles from figma, each value is inserted inside a subfolder with the same name as the token category
Example in the screenshot
Instead of importing letterSpacing-0 inside the 'Letter Spacing' folder, it creates a letterSpacing folder with a token named '0' inside the Letter Spacing folder, so we have an extra unnecessary nested folder as well as conflicting token names

**To Reproduce**
Open plugin
Click Styles at the bottom
Click Import styles

**Expected behavior**
Should insert key/value pairs inside the proper 'type' folder, without creating a subfolder named 'type'

**Screenshots or Screencasts**
<img width="634" alt="image" src="https://user-images.githubusercontent.com/75637000/205902435-ba22ee43-fa74-4c17-a432-b3671909abb5.png">

**Figma file (optional)**
Send us the link to a Figma file containing a reproducible example

**JSON (optional)**
 "letterSpacing": {
      "0": {
        "value": "0%",
        "type": "letterSpacing"
      }
    },
    "paragraphSpacing": {
      "0": {
        "value": "0",
        "type": "paragraphSpacing"
      }
    },
    
    etc...

Basically it is wrapped inside an unnecessary subfolder while the key: {value, type} is sufficient to position it inside the proper category folder

**Comment by six7** [comment]
This isn't really a problem related to how things are imported, but how things are displayed.

The `type folder` you mentioned isn't a folder, it's a grouping by `type of token`, so `fontFamily` shows all tokens that are of `type: fontFamily`. Token names are unique across all types, which is why that second nesting is required (and I agree it doesn't make a lot of sense in the current grouping!)

We'll address this with a different way to group things in a coming release. We'll use this issue to track work: https://github.com/tokens-studio/figma-plugin/issues/1024

If I misunderstood let me know and we'll reopen! 🙏 But I feel that issue would solve it for you.
